docs/tut-making-shooter.md

```diff
@@ -190,7 +190,7 @@ This line reads built-in variables and moves the Copy according to them. Without
 
 There are more built-in variables, which you can find on the [`ct.templates` page](ct.templates.html).
 
-We will modify the `Step` code so enemies will destroy themselves if they fall off the screen.
```

Comment:
uhm, the code here is actually outdated @_@
ct.viewHeight / ct.viewWidth no longer exist, their equivalents are ct.camera.width and ct.camera.height